File: event-bus-rabbit/src/main/java/pegasus/eventbus/topology/event/GetEventTypeRoute.java
package pegasus.eventbus.topology.event;

public class GetEventTypeRoute {

	private String eventTypeCannonicalName;

    //@todo - needed for gson in osgi
	public GetEventTypeRoute() {
	    
	}
	
	public GetEventTypeRoute(String eventTypeCannonicalName) {
		super();
		this.eventTypeCannonicalName = eventTypeCannonicalName;
	}

	public String getEventTypeCanonicalName() {
		return eventTypeCannonicalName;
	}
	
	public void setEventTypeCanonicalName(String eventTypeCannonicalName) {
	    this.eventTypeCannonicalName = eventTypeCannonicalName;
	}
	
}

File: event-bus-rabbit/src/main/dotnet/topology/events/GetEventTypeRoute.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace pegasus.eventbus.topology.events
{
    public class GetEventTypeRoute
    {
        public string EventTypeCannonicalName;

        public GetEventTypeRoute() { }
        public GetEventTypeRoute(string eventType) { this.EventTypeCannonicalName = eventType; }
    }
}
